docs/subvolume.md

```diff
@@ -0,0 +1,53 @@
+# Subvolume cleanup
+
+The subvolume command is used to clean the stale subvolumes
+which have no parent-pvc attached to them.
+The command would list out all such subvolumes which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
+
+The subvolume command will require the following sub commands:
+* `ls` : [ls](#ls) lists all the subvolumes
+    * `--type=stale`: lists all the stale volume that can be deleted
+* `delete <subvolumes> <filesystem> <subvolumegroup>`:
+    [delete](#delete) stale subvolumes after user's confirmation.
+    It will list and delete only the stale subvolumes to prevent any loss of data.
+    * subvolumes: comma-separated list of subvolumes of same filesystem and subvolumegroup.
```

Comment:
```suggestion
The subvolume command will require the following sub commands:
* `ls` : [ls](#ls) lists all the subvolumes
    * `--type=stale`: lists all the stale volume that can be deleted
* `delete <subvolumes> <filesystem> <subvolumegroup>`:
    [delete](#delete) stale subvolumes after user's confirmation.
    It will list and delete only the stale subvolumes to prevent any loss of data.
    * subvolumes: comma-separated list of subvolumes of same filesystem and subvolumegroup.
```

Please update the doc similar to this https://github.com/red-hat-storage/odf-cli/blob/cfac47a3970e0177cbded34b1544b9c2bc5e76dc/docs/set.md